### ISSUE TITLE
More detailed installation instructions?

### ISSUE THREAD

**Issue by Ahti** [Qusic/atom-youcompleteme#15]
I'm trying to install you-complete-me. I installed the package from inside Atom. When the package is loaded, I get two notifications in the Atom window:

![screen shot 2015-06-19 at 17 35 51](https://cloud.githubusercontent.com/assets/644764/8256825/dcf792ec-16a9-11e5-9040-491181d3ecab.png)

Trying to follow the instructions in the readme, I found that `~/.atom/packages/you-complete-me/ycmd` does not exist.

Is there some additional step I need to take to install this? If so, it would probably be good to document that in the readme.

**Comment by Qusic** [comment]
It should automatically download ycmd from github. I'm not sure what's wrong with your setup. Can you check the console and provide more details on the error?

**Comment by Qusic** [comment]
You can also download ycmd yourself.

This is what the script do:
```
cd ~/.atom/packages/you-complete-me/
git clone https://github.com/Qusic/ycmd.git
cd ycmd
git checkout 5796f7d3bccd7050384c4cc710845351bcd416e2
```

Then you just follow the Usage in README.

I am quite busy recently at school and will update the documentation later in my summer vacation.

**Comment by Ahti** [comment]
When I run `atom -df`, this is the console output:

```
2015-06-19 18:04:06.958 Atom[3560:98894] App load time: 263ms
[3560:0619/180407:INFO:CONSOLE(43)] "activate linter-clang", source: /Users/lukas/.atom/packages/linter-clang/lib/init.coffee (43)
[3560:0619/180407:INFO:CONSOLE(11)] "activate linter-lua", source: /Users/lukas/.atom/packages/linter-lua/lib/init.coffee (11)
[3560:0619/180408:INFO:CONSOLE(56)] "Window load time: 997ms", source: file:///Applications/Atom.app/Contents/Resources/app.asar/static/index.js (56)
[3560:0619/180408:INFO:CONSOLE(9)] "[YCM-CMD]", source: /Users/lukas/.atom/packages/you-complete-me/lib/debug.coffee (9)
[3560:0619/180408:INFO:CONSOLE(9)] "[YCM-CMD]", source: /Users/lukas/.atom/packages/you-complete-me/lib/debug.coffee (9)
[3560:0619/180409:INFO:CONSOLE(9)] "[YCM-CMD]", source: /Users/lukas/.atom/packages/you-complete-me/lib/debug.coffee (9)
[3560:0619/180409:INFO:CONSOLE(14)] "Unhandled promise rejection %o with error: %o", source: file:///Applications/Atom.app/Contents/Resources/app.asar/static/index.js (14)
```

(I left everything in, even stuff that didn't seem relevant to me)

**Comment by Qusic** [comment]
Oh sorry. I looked through the code just now and it was a typo. Just published v0.4.1 and it should work now.
